### ISSUE TITLE
Add json-rpc guidance info for users of unsupported programming languages

### ISSUE THREAD

**Issue by michzio** [ethereum/ethereum-org-website#2222]
Is there Etherum for Swift developers like other projects kind of Etherum for Python developers?

**Comment by wackerow** [comment]
@michzio Hey, glad to hear you're interested in getting involved! Unfortunately we don't currently have any resources for Swift programmers specifically.

@samajammin Any idea best place to refer, or if you're aware of any Swift resources that we should consider incorporating into the site?

**Comment by qbzzt** [comment]
To use Ethereum from an unsupported language, such as Swift, you connect to a client (a node in the Ethereum network) directly using JSON-RPC, as documented here: https://eth.wiki/json-rpc/API.

**Comment by wackerow** [comment]
@qbzzt Good call, thanks for suggesting this... @michzio I'm going to close this for now, please feel free to hop over to [our Discord](https://discord.gg/CetY6Y4) to chat more about this informally =)

**Comment by qbzzt** [comment]
Maybe we should have a note in https://ethereum.org/en/developers/docs/programming-languages saying that any unsupported languages can use Ethereum using JSON-RPC?

**Comment by wackerow** [comment]
@qbzzt I'll reopen using that as the new issue... Would welcome a PR for this =) Lemme know if you're down, I'll assign ya. 

Just a short blurb on the `programming-languages` docs page helping direct anyone using languages not discussed would be nice.

**Comment by qbzzt** [comment]
I think I'm up to it, assign me please.

**Comment by qbzzt** [comment]
Once #2631 is merged we can close this issue.
